packages/lab/src/Flow/stories/SubFlow/Level.styles.tsx

```diff
@@ -0,0 +1,43 @@
+import { createClasses, theme } from "@hitachivantara/uikit-react-core";
+
+export const { staticClasses, useClasses } = createClasses("HvLevel", {
```

Comment:
Since this is a story and not one of our components, maybe we should drop `createClasses` and create classes as we usually do for stories (using `css`). I don't think we need this "overhead" for the sample.

Also, I didn't find where is file is used. Is it needed?

Reply:
Leftover, It was not supposed to be on the commit, going to remove it.